unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioSource.cs

```diff
@@ -113,7 +113,7 @@ private void OnCurrentSceneChanged(string currentSceneId, string previousSceneId
         {
             if (audioSource != null)
             {
-                audioSource.volume = (scene.sceneData.id == currentSceneId) ? model.volume : 0f;
+                audioSource.volume = ((scene.sceneData.id == currentSceneId) || (scene is GlobalScene && ((GlobalScene)scene).isPortableExperience)) ? model.volume : 0f;
```

Comment:
can we change this to make it a little bit more clear? is a very long line :)
```suggestion
                float volume = 0;
                if ((scene.sceneData.id == currentSceneId) || (scene is GlobalScene globalScene && globalScene.isPortableExperience)){
                    volume =  model.volume;
                }
                audioSource.volume = volume;
```

Reply:
Great, I think your approach is clearer, thanks! Done!